.azure/hpu-tests.yml

```diff
@@ -51,6 +51,16 @@ jobs:
         sudo pip install pip -U
       displayName: 'Instance HW info'
 
+    - bash: |
+        pip install -e . --find-links ${TORCH_URL}
+        git checkout -- setup.py MANIFEST.in
+      env:
+        PACKAGE_NAME: "lite"
+        FREEZE_REQUIREMENTS: "1"
+      # Lite shall have pin version, so to keep development aligned we need to install lite from source not from pypi
+      # also installing from source not from on-the-fly created package tp prevent accidental interaction with cache
```

Comment:
typo
```suggestion
      # also installing from source not from on-the-fly created package to prevent accidental interaction with cache
```

Reply:
Just to check my understanding, the "on-the-fly created package" mean the lite package pulled from pypi, right?